### PR TITLE
Updating publish transforms to work with multiple dw_types

### PR DESCRIPTION
**Pull request by ChrisGriffithRASGO**
Adding support to publish transforms for each of the current supported DWs in RasgoAPI, as well as a generic one. Dependent on RasgoAPI and RasgoSDK changes.

(Seems like there are some pre-commit things that are not being run for this repo, so there is also some cleanup there. Need to figure out why pre-commit.ci isn't working)

### CODE REVIEW

File: python/publish_transforms.py [hunk 1]
```diff
@@ -30,7 +30,12 @@ def publish_transforms(rasgo_api_key: str, rasgo_domain: str) -> None:
 
     # Get all Rasgo Transforms
     rasgo = pyrasgo.connect(rasgo_api_key)
-    rasgo_transforms = utils.get_all_rasgo_transform_keyed_by_name(rasgo)
+
+    org = rasgo.get.organization()
+    if org.name != "Community":
```
Comment (griffatrasgo) [inline python/publish_transforms.py:35]: should we use something like org.id == 1 (and/or make COMMUNITY_ORG_ID = 1 an int in constants.py we can ref)?

Comment (griffatrasgo) [reply python/publish_transforms.py:35]: Reason being: in the future users may be able to change their org name, but we control the id (and 1 is consistent across our envs)

**Review by griffatrasgo — APPROVED** [review]

File: python/constants.py [hunk 2]
```diff
@@ -21,3 +21,5 @@ class PyRasgoEnvironment(Enum):
 # TODO: When we support multiple DWs in the API
 # we'll need to refactor the functions that consume this
 RASGO_DATAWAREHOUSE = 'snowflake'
+
+COMMUNITY_ORGANIZATION_ID: int = 1
```
Comment (szaghikian-rasgo) [inline python/constants.py:25]: I'm loving that this is going in as a var, but I wonder if it should be an enum; it's possible that we have diff org IDs we want to publish with by environment (if not now, potentially in the future). Not sure that it blocks right now, tho

**Review by szaghikian-rasgo — APPROVED** [review]
Looks good, left a comment about community org ID as a var vs as an enum driven by env.

File: python/constants.py (hunk 2 above)
Comment (ChrisGriffithRASGO) [reply python/constants.py:25]: May not to be bad to add if we ever need it, but as this is just our utils for now, I'm not too worried about perfect structure / plan for future as we might nix this all in a few months

**Pull request merged**